include/utility.hpp

```diff
@@ -303,6 +305,46 @@ template <class CB_CORE, typename E> class TEnumInfo {
     }
     info.labels.elements = &clabels[0];
     info.labels.len = uint32_t(labels.size());
+
+    for (auto &v : vseq) {
+      values.emplace_back(CBEnum(v));
```

Comment:
If you use emplace your call is already a constructor call. So CBEnum should be redundant in this case.

Reply:
Doesn't compile with some cryptic errors if I don't put that cast.
> C:/Tools/msys64/mingw64/include/c++/10.3.0/ext/new_allocator.h:150:4: error: cannot convert 'const chainblocks::Math::Mean::MeanKind' to 'int' in initialization